utils/testrunner/TestRunner.js

```diff
@@ -422,7 +395,10 @@ class TestRunner extends EventEmitter {
     const runnableTests = this._runnableTests();
     this.emit(TestRunner.Events.Started, runnableTests);
     this._runningPass = new TestPass(this, this._rootSuite, runnableTests, this._parallel, this._breakOnFailure);
-    const termination = await this._runningPass.run();
+    const termination = await this._runningPass.run().catch(e => {
+      console.error(e);
```

Comment:
this looks strange? We log an error and then throw it again? Is something consuming errors or something? Leftover debug code?

Reply:
if `testrunner.run()` is not awaited and there's an issue with testrunner itself, then it crashes silently with an unhandled promise rejection that doesn't have any stack.

This gives the stack.

Reply:
Is there a case where this gives you a double error message? Or this is always just problems with the test runner itself, to avoid unhandledpromiserejection woes?

Reply:
Yeah, that's always for the bugs in the testrunner itself